Classes/Protecta.uc

```diff
@@ -14,41 +14,6 @@ exec function SwitchModes()
     DoToggle();
 }
 
-simulated function bool CanZoomNow()
-{
-    // aiming while firing makes zoom bugged. Prevent zooming while firing.
-    return Instigator != none && Instigator.Controller.bFire == 0;
```

Comment:
Are you sure this isn't needed anymore? This is the third attempt to get fix zooming bug :)

Reply:
the zooming bug was caused by bad idle positions and bad recoil animations, both are fixed in the new animation package so this isn't needed anymore

Reply:
OK, we'll see :smile: